### PR TITLE
feat: capture performance opt in

### PR DESCRIPTION
**Pull request by pauldambra**
## Problem

safe to merge but should wait for #13299

set up for enabling capturing performance alongside enabling performance logs

## Changes

optin

## How did you test this code?

👀

### CODE REVIEW

File: frontend/src/scenes/session-recordings/settings/SessionRecordingSettings.tsx [hunk 1]
```diff
@@ -69,6 +71,28 @@ export function SessionRecordingSettings({ inModal = false }: SessionRecordingSe
...
+                        data-attr="opt-in-capture-performance-switch"
+                        onChange={(checked) => {
+                            updateCurrentTeam({ capture_performance_opt_in: checked })
+                        }}
+                        label="Capture network requests and performance information within recordings"
+                        labelClassName={inModal ? 'text-base font-semibold' : ''}
+                        bordered={!inModal}
+                        fullWidth={inModal}
+                        checked={!!currentTeam?.capture_performance_opt_in}
+                        disabled={!currentTeam?.session_recording_opt_in}
```
Comment (benjackwhite) [inline frontend/src/scenes/session-recordings/settings/SessionRecordingSettings.tsx:86]: I think we could just remove this disabled line as the performance capture is not tightly coupled in the JS

Comment (pauldambra) [inline frontend/src/scenes/session-recordings/settings/SessionRecordingSettings.tsx:86]:
```suggestion
```

**Review by benjackwhite — APPROVED** [review]

**Review by benjackwhite — CHANGES_REQUESTED** [review]
Started fixing the test and realised - we should not include the web performance as part of recordings. It limits us too much for the future. I'm going to move it out to a top-level property

**Review by benjackwhite — APPROVED** [review]
I approve of my changes

**Pull request merged**